graph/graph.go

```diff
@@ -112,11 +113,18 @@ func NewGraph(root string, driver graphdriver.Driver) (*Graph, error) {
 	}
 
 	graph := &Graph{
-		root:     abspath,
-		idIndex:  truncindex.NewTruncIndex([]string{}),
-		driver:   driver,
-		retained: &retainedLayers{layerHolders: make(map[string]map[string]struct{})},
+		root:             abspath,
+		idIndex:          truncindex.NewTruncIndex([]string{}),
+		driver:           driver,
+		retained:         &retainedLayers{layerHolders: make(map[string]map[string]struct{})},
+		tarSplitDisabled: false,
 	}
+
+	// Windows does not currently support tarsplit functionality.
+	if runtime.GOOS == "windows" {
+		graph.tarSplitDisabled = true
```

Comment:
Yeah, we'd certainly appreciate the help.  I had an email ready to contact you about it, but then we decided it would be better to unblock progress for the short term by skipping it since the resolution is likely to be complicated.  Our transport format (which gets produced during the call to Diff) is different than the on-disk layer format.  I think that is part of the complication.